communication/slack-config/users.yaml

```diff
@@ -15,9 +17,8 @@ users:
   jeefy: U5MCFK468
   justaugustus: U0E0E78AK
   katharine: UBTBNJ6GL
-  listx: UFCU8S8P3
```

Comment:
There's no need to ever remove people from this list. Please leave them in for potential future use.

Reply:
SGTM. Backed out that change.